src/Bubble.js

```diff
@@ -161,6 +161,21 @@ const styles = {
       borderTopLeftRadius: 3,
     },
   }),
+  center: StyleSheet.create({
+    container: {
+      flex: 1,
```

Comment:
@dgdavid I checked both platform and `flex: 1` is unnecessary (I have already removed `flex: 1` from Bubble).